PoetrySlam/api/v1/endpoints/poetryslam_routes.py

```diff
@@ -13,3 +13,7 @@ def read_root():
 @router.get("/items/{item_id}")
 def read_item(item_id: int, q: Optional[str] = None):
     return {"item_id": item_id, "q": q}
+
+@router.get("/name/")
+def read_name(first_name: str, last_name: str):
+    return {"first_name": first_name, "last_name": last_name}
```

Comment:
needs a newline at the end of file otherwise it will fail linting.

`poetry run pycodestyle PoetrySlam`
this command should return nothing

I will be adding the ci pipeline via github actions so it can be done automatically